tools/bcftools_filter_index.cwl

```diff
@@ -0,0 +1,113 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: bcftools_reheader_sort
+doc: |
+  BCFTOOLS reheader and sort
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram*1000)
+    coresMin: $(inputs.cpu)
+  - class: DockerRequirement
+    dockerPull: 'staphb/bcftools:1.10.2'
+baseCommand: []
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >
+      bcftools filter
```

Comment:
Shouldn't the command be `bcftools reheader`, given the name of the tool? If you really have a "jack of all trades" tool, you can just name the steps differently and have only one tool

Reply:
The ID and description are actually wrong here. The tool is for filtering and indexing. Will change before merge.